3 - PB/Documentazione interna/Verbali/24-03-17/24-03-17.typ

```diff
@@ -0,0 +1,105 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "17/03/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+
+  timeStart: "15:00",
+  timeEnd: "16:00",
+);
+
+= Ordine del giorno
+
+- Valutazione del progresso generale;
+- Analisi retrospettiva;
+- Data di consegna;
+- Pianificazione.
+
+= Valutazione del progresso generale <avanzamento>
+
+Lo Sprint 19 ha visto buona parte dei task assegnati, sia riguardanti la documentazione che di codifica, completati nei tempi previsti. I task non conclusi risultano in corso d'opera e verranno portati in verifica nel prossimo Sprint.
+
+Il giorno venerdì 15/03/2024 si è svolto il meeting esterno con il Proponente, come pianificato, per un aggiornamento riguardo lo stato dei lavori e gli avanzamenti nello sviluppo dell'MVP.
+
+== #adr
+
+Gli Use Case sono stati estesi e i requisiti aggiornati in modo da rispecchiare con maggiore precisione le richieste del Proponente.
+
+== #ndp
+
+Sono state redatte e verificate le sezioni "Validation process" e "Transition process".
+
+== #pdp
+
+È stato redatto il consuntivo di periodo relativo allo Sprint 18.
+
+== #st
+
+Sono state redatte le sezioni relative alla:
+- struttura dell'applicazione WMS3;
+- data pattern utilizzati;
+- classi e componenti presenti in ciascun layer architetturale (con i corrispondenti diagrammi delle classi).
+
+Inoltre è stata aggiunta la sezione riguardante l'installazione.
+
+== Codifica
+
+L'attività di codifica ha visto completati i lavori:
+- realizzazione della creazione del piano rettangolare in Three.js;
+- implementazione della lettura del file SVG;
+- implementazione dell'elemento "Order Panel Item";
+- implementazione del pulsante "reimposta" nel Settings Panel;
+- implementazione della creazione di una zona;
+- implementazione della modifica di una zona;
+- implementazione dell'eliminazione di una zona;
+- rimozione del pulsante "sincronizzazione" da Order Panel;
+- implementazione del pulsante "sincronizzazione" in Settings Panel;
+- implementazione del componente "Prodotto Details";
+- implementazione "Product Item" per il Product Panel;
+- implementazione dei campi di modifica delle dimensioni della planimetria in Settings Panel;
+- implementazione di una simulazione delle API per lo spostamento dei prodotti;
+- Implementazione test per le API prodotte.
```

Comment:
Lo cambierei in "implementazione test per le Server Actions": la/il task si riferiva alle API perché quando è stata fatta la pianificazione avevamo ancora in mente di usare le API